### ISSUE TITLE
[dev.icinga.com #6643] Import Performance Data (broken?)

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga2#1756]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/6643

**Created by bsheqa on 2014-07-04 13:51:27 +00:00**

Assignee: _(none)_
Status: _Rejected_  (closed on _2015-01-12 09:34:01 +00:00_)
Target Version: _(none)_
Last Update: _2015-01-12 09:34:01 +00:00 (in Redmine)_

    Icinga Version: r2.0.0-3

---

I am experiencing problems when importing historical performance data using the "PROCESS\_FILE" command.

Command I am using:
`` echo "[`date +%s`] PROCESS_FILE;/usr/local/src/host-perfdata;0" >> /var/run/icinga2/cmd/icinga2.cmd ``

Here's a snippet from the host-perfdata file:
@
\[1403789550\] PROCESS\_HOST\_CHECK\_RESULT;c2-mysql-1;1;PING CRITICAL - Packet loss = 100%|
\[1403789550\] PROCESS\_HOST\_CHECK\_RESULT;c2-switch-windows;0;PING OK - Packet loss = 0%, RTA = 0.07 ms|rta=0.064000ms;3000.000000;5000.000000;0.000000
\[1403789580\] PROCESS\_HOST\_CHECK\_RESULT;c1-linux-1;0;PING OK - Packet loss = 0%, RTA = 0.07 ms|rta=0.059000ms;3000.000000;5000.000000;0.000000
\[1403789580\] PROCESS\_HOST\_CHECK\_RESULT;hp-procurve-switch;0;PING OK - Packet loss = 0%, RTA = 0.07 ms|rta=2.327000ms;3000.000000;5000.000000;0.000000
\[1403789590\] PROCESS\_HOST\_CHECK\_RESULT;c1-linux-2;0;PING OK - Packet loss = 0%, RTA = 0.07 ms|rta=0.052000ms;3000.000000;5000.000000;0.000000
\[1403789590\] PROCESS\_HOST\_CHECK\_RESULT;hw-router;0;PING OK - Packet loss = 0%, RTA = 0.07 ms|rta=0.030000ms;3000.000000;5000.000000;0.000000
\[1403789610\] PROCESS\_HOST\_CHECK\_RESULT;c1-linux-3;0;PING OK - Packet loss = 0%, RTA = 0.07 ms|rta=0.054000ms;3000.000000;5000.000000;0.000000
\[1403789610\] PROCESS\_HOST\_CHECK\_RESULT;hw-switch-hardware;0;PING OK - Packet loss = 0%, RTA = 0.07 ms|rta=0.050000ms;3000.000000;5000.000000;0.000000
@

Expected behaviour:
Historical data is being imported by using the timestamp provided.

Actual behaviour:
States of hosts change, but just with the current timestamp. not with the one provided in the host-perfdata file.

Additional Information:
'checker'-feature of icinga2 is disabled.

Please find the icinga configuration I am using attached below.

**Attachments**

* [icinga2-config.tar.gz](https://dev.icinga.com/attachments/download/1875/icinga2-config.tar.gz) bsheqa - _2014-07-04 13:50:26 +00:00_
